recipes/cyclonedds/all/patches/0.10.2-0001-fix-find-iceoryx.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
+index fa08094a..516806dd 100644
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -55,8 +55,8 @@ if(ENABLE_SHM)
+   else()
+     set(iceoryx_required QUIET)
+   endif()
+-  find_package(iceoryx_binding_c ${iceoryx_required})
+-  set(ENABLE_SHM ${iceoryx_binding_c_FOUND} CACHE STRING "" FORCE)
++  find_package(iceoryx ${iceoryx_required})
```

Comment:
iceoryx is just required in case you activate the conan option "with_shm" (with shared memory) 
cyclonedds then require iceoryx but otherwise not - thats why i didnt patch with "REQUIRED".

Reply:
I see, yes, than it makes sense keeping optional